RUFAS/routines/animal/manure/general_manure.py

```diff
@@ -5,53 +5,58 @@
 
 
 class AnimalManureExcretions(TypedDict):
-    """A TypedDict class that specifies the structure of the dictionary of animal manure excretion values.
+    """A TypedDict class that specifies the structure of the dictionary of animal manure excretion values."""
 
-    """
     urea: float
-    """Concentration of urea in manure, g/L."""
+    """Concentration of urea in manure (g/L)."""
 
     urine: float
-    """Amount of urine excreted, kg."""
+    """Amount of urine excreted (kg)."""
 
     total_ammoniacal_nitrogen_concentration: float
-    """Concentration of total ammoniacal manure_nitrogen in the manure slurry, g/L."""
+    """Concentration of total ammoniacal manure_nitrogen in the manure slurry (g/L)."""
 
     urine_nitrogen: float
-    """Amount of nitrogen in urine, kg."""
+    """Amount of nitrogen in urine (kg)."""
 
     manure_nitrogen: float
-    """Amount of nitrogen in manure, kg."""
+    """Amount of nitrogen in manure (kg)."""
 
     manure_mass: float
-    """Amount of manure, kg."""
+    """Amount of manure (kg)."""
 
     total_solids: float
-    """Amount of total solids, kg."""
+    """Amount of total solids (kg)."""
 
     degradable_volatile_solids: float
-    """Amount of degradable volatile solids, kg."""
+    """Amount of degradable volatile solids (kg)."""
 
     non_degradable_volatile_solids: float
-    """Amount of non-degradable volatile solids, kg."""
+    """Amount of non-degradable volatile solids (kg)."""
 
     inorganic_phosphorus_fraction: float
-    """Fraction of water extractable inorganic phosphorus, unitless."""
+    """Fraction of water extractable inorganic phosphorus (unitless)."""
 
     organic_phosphorus_fraction: float
-    """Fraction of water extractable organic phosphorus, unitless."""
+    """Fraction of water extractable organic phosphorus (unitless)."""
+
+    non_water_inorganic_phosphorus_fraction: float
```

Comment:
```suggestion
    non_water_extractable_inorganic_phosphorus_fraction: float
```
Should we be as explicit as possible here?

Would it be a good idea to rename `inorganic_phosphorus_fraction` and `organic_phosphorus_fraction` to `water_extractable_inorganic_phosphorus_fraction` and `water_extractable_organic_phosphorus_fraction`?

Reply:
I'm ok with either choice. The current names seem to be long enough, though. If we decide to make these two fully explicit, we may need to do the same for the other two existing water-extractable variables to be consistent. All of this can be in a separate PR.